bin/config.py

```diff
@@ -0,0 +1,6 @@
+"""Configuration file for server.py"""
+import os
+
+PORT = os.environ.get('PORT')
+MEDIAWIKI_URL = os.environ.get('MEDIAWIKI_URL')
+CLIENT_URL = os.environ.get('CLIENT_URL')
```

Comment:
Let's rename it to `ALLOWED_HOSTS` and we will make available requests from several clients.